www/src/components/StickyResponsiveSidebar/StickyResponsiveSidebar.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails react-core
+*/
+
+'use strict';
+
+import Container from 'components/Container';
+import {Component, React} from 'react';
+import isItemActive from 'utils/isItemActive';
+import {Sticky} from 'react-sticky';
+import Sidebar from 'templates/components/Sidebar';
+import {colors, media} from 'theme';
+
+// TODO: memoize to save doing O(n) on the active section items + subitems every
+// time.
+function findActiveItemTitle(location, defaultActiveSection) {
+  const {items} = defaultActiveSection;
```

Comment:
This line causes a build error for me:
> WebpackError: Cannot read property 'items' of null

Reply:
Note that I happened to have this page open when running `yarn dev`:
http://localhost:8000/blog/2013/06/12/community-roundup.html